ptf/saiswitch.py

```diff
@@ -224,29 +254,24 @@ def availableIPv6RouteEntryTest(self):
                     route_number += 1
                 elif status == SAI_STATUS_ITEM_ALREADY_EXISTS:
                     continue
-                elif mask == '/128':  # when host table is full change to LPM
+                elif mask == '/32':  # when host table is full change to LPM
                     print("%s host routes have been created" % route_number)
                     max_host_route = route_number
-                    self.available_v6_host_routes = max_host_route
-                    mask = '/120'
-                    continue
-                elif mask == '/120':  # when LPM table is full change to LPM64
-                    print("%s host + LPM routes have been created so far"
-                          % route_number)
-                    mask = '/64'
+                    self.available_v4_host_routes = max_host_route
+                    mask = '/30'
                     continue
                 else:
                     self.fail("Route creation failed after creating %d "
                               "entries, status %u" % (route_number, status))
 
                 attr = sai_thrift_get_switch_attribute(
-                    self.client, available_ipv6_route_entry=True)
-                self.assertEqual(attr["available_ipv6_route_entry"],
+                    self.client, available_ipv4_route_entry=True)
+                self.assertEqual(attr["available_ipv4_route_entry"],
                                  max_route_entry - route_number)
 
```

Comment:
This test case may lack a logic that if the host table is not full until the loop ends, we may assign the value to`self.available_v4_host_routes` which is used in test case `self.availableIPv4NeighborEntryTest()`
```python
self.available_v4_host_routes = route_number
```

Reply:
test cases method will be split into smaller classes, and those variables will be assigned in the new test class.